guides/doc-Upgrading_and_Updating/topics/updating_satellite_server_to_next_minor_version.adoc

```diff
@@ -103,54 +98,155 @@ For more information about configuring download policies, see {ContentManagement
 
 [{RepoRHEL7ServerAnsible}]
 name=Ansible {SatelliteAnsibleVersion} RPMs for Red Hat Enterprise Linux 7 Server x86_64
-baseurl=file:///var/lib/pulp/published/yum/https/repos/My_Organization/Library/content/dist/rhel/server/7/7Server/x86_64/ansible/2.9/os/
+baseurl=https://{foreman-example-com}/pulp/content/My_Organization/Library/content/dist/rhel/server/7/7Server/x86_64/ansible/2.9/os/
```

Comment:
Why hardcode x86_64 and not use `$basearch`? Also, instead of `7Server` I believe you can write `$releasever` in yum repo files. Sadly I don't think there's a macro in RHEL that resolves to `7` so you can't make the path fully dynamic.
```suggestion
baseurl=https://{foreman-example-com}/pulp/content/My_Organization/Library/content/dist/rhel/server/7/$releasever/$basearch/ansible/{SatelliteAnsibleVersion}/os/
```

This should get you closer to preparing for supporting RHEL 8.